src/Plugins/RpcServer/RpcServer.cs

```diff
@@ -261,24 +263,83 @@ public async Task ProcessAsync(HttpContext context)
         private async Task<JObject> ProcessRequestAsync(HttpContext context, JObject request)
         {
             if (!request.ContainsProperty("id")) return null;
-            JToken @params = request["params"] ?? new JArray();
+            var @params = request["params"] ?? new JArray();
             if (!request.ContainsProperty("method") || @params is not JArray)
             {
                 return CreateErrorResponse(request["id"], RpcError.InvalidRequest);
             }
-            JObject response = CreateResponse(request["id"]);
+
+            var jsonParameters = (JArray)@params;
+            var response = CreateResponse(request["id"]);
             try
             {
-                string method = request["method"].AsString();
+                var method = request["method"].AsString();
                 (CheckAuth(context) && !settings.DisabledMethods.Contains(method)).True_Or(RpcError.AccessDenied);
-                methods.TryGetValue(method, out var func).True_Or(RpcErrorFactory.MethodNotFound(method));
-                response["result"] = func((JArray)@params) switch
+
+                if (methods.TryGetValue(method, out var func))
+                {
+                    response["result"] = func(jsonParameters) switch
+                    {
+                        JToken result => result,
+                        Task<JToken> task => await task,
+                        _ => throw new NotSupportedException()
+                    };
+                    return response;
+                }
+
+                if (_methodsWithParams.TryGetValue(method, out var func2))
                 {
-                    JToken result => result,
-                    Task<JToken> task => await task,
-                    _ => throw new NotSupportedException()
-                };
-                return response;
+                    var paramInfos = func2.Method.GetParameters();
+                    var args = new object[paramInfos.Length];
+
+                    for (var i = 0; i < paramInfos.Length; i++)
+                    {
+                        var param = paramInfos[i];
+                        if (jsonParameters.Count > i && jsonParameters[i] != null)
+                        {
+                            try
+                            {
+                                if (param.ParameterType == typeof(UInt160))
+                                {
+                                    args[i] = ParameterConverter.ConvertUInt160(jsonParameters[i], system.Settings.AddressVersion);
+                                }
+                                else
+                                {
+                                    args[i] = ParameterConverter.ConvertParameter(jsonParameters[i], param.ParameterType);
+                                }
```

Comment:
Shouldn't we also try `ParameterConverter.ConvertUInt256` if the parameter type is UInt256?

Reply:
UInt256 will be processed in ConvertParameter, UInt160 is special cause it needs `AddressVersion ` while all other types do not need it.